src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = !century
+    ? people.filter((person) => person.sex === 'm')
+    : people.filter((person) => {
+      return person.sex === 'm' && Math.ceil(person.died / 100) === century;
+    });
```

Comment:
Try to look deeper into this part. You have two repeated equal conditions:
`person.sex === 'm'`
It's because at first you need all men to be filtered from list. Another condition depends on `century` parameter.
Just filter all person by gender **and** if century is defined - filter by century.
```
people.filter(person => {
  return <by gender is 'm'> && (<if century ? by century is same : in other case simple true>)
})
```